plone/app/content/browser/contents/templates/properties.pt

```diff
@@ -1,63 +1,116 @@
-    <textarea class="form-control" name="copyright"></textarea>
+    <label class="form-label"
+           i18n:translate="copyright"
+    >Copyright</label>
+    <textarea class="form-control"
+              name="copyright"
+    ></textarea>
   </div>
 
   <div class="mb-2">
-    <label class="form-label" i18n:translate="creators">Creators</label>
-    <input name="creators" class="form-control"
-           class="pat-select2"
+    <label class="form-label"
+           i18n:translate="creators"
+    >Creators</label>
+    <input class="pat-select2"
+           name="creators"
            data-pat-select2="multiple: true;
-                             vocabularyUrl: ${python: options['vocabulary_url']}"/>
+                             vocabularyUrl: ${python: options['vocabulary_url']}"
+    />
   </div>
 
   <div class="mb-2">
-    <label class="form-label" i18n:translate="contributors">Contributors</label>
-    <input name="contributors" class="form-control"
-           class="pat-select2"
+    <label class="form-label"
+           i18n:translate="contributors"
+    >Contributors</label>
+    <input class="pat-select2"
+           name="contributors"
            data-pat-select2="multiple: true;
-                             vocabularyUrl: ${python: options['vocabulary_url']}"/>
+                             vocabularyUrl: ${python: options['vocabulary_url']}"
+    />
   </div>
 
   <div class="mb-2">
-    <label class="form-label" for="fcSwitchExcludeFromNav" i18n:translate="exclude_from_nav">Exclude from navigation</label>
+    <label class="form-label"
+           for="fcSwitchExcludeFromNav"
+           i18n:translate="exclude_from_nav"
+    >Exclude from navigation</label>
     <div class="form-check">
-      <input class="form-check-input" type="radio" name="exclude-from-nav" id="fcSwitchExcludeFromNavYes" value="yes">
-      <label class="form-check-label" for="fcSwitchExcludeFromNavYes" i18n:translate="yes">Yes</label>
+      <input class="form-check-input"
+             id="fcSwitchExcludeFromNavYes"
+             name="exclude-from-nav"
+             type="radio"
+             value="yes"
+      />
+      <label class="form-check-label"
+             for="fcSwitchExcludeFromNavYes"
+             i18n:translate="yes"
+      >Yes</label>
     </div>
     <div class="form-check">
-      <input class="form-check-input" type="radio" name="exclude-from-nav" id="fcSwitchExcludeFromNavNo" value="no">
-      <label class="form-check-label" for="fcSwitchExcludeFromNavNo" i18n:translate="no">No</label>
+      <input class="form-check-input"
+             id="fcSwitchExcludeFromNavNo"
+             name="exclude-from-nav"
+             type="radio"
+             value="no"
+      />
+      <label class="form-check-label"
+             for="fcSwitchExcludeFromNavNo"
+             i18n:translate="no"
+      >No</label>
     </div>
   </div>
 
-  <% if (data.languages) { %>
+  &lt;% if (data.languages) { %&gt;
```

Comment:
I quickly scrolled through the template changes, and this stood out as suspicious.  But I don't know what the original `<%` does.  I have not seen this construction.  There are a few more like these in other templates.

Could you check if this still works?

Reply:
That's why I was not a huge fan of applying `zpretty` but let's see... I will double check it.

At least tests-wise they all run 🤞🏾

Reply:
@ale-rt it should not touch such instructions this way? Is this intentional or a bug?

Reply:
IMO `zpretty` does the right thing by fixing the XML.

It seems to me that chameleon does not even know what that syntax is:
```
[ale@flo chameleon]$ cat c.pt 
  <% if (data.languages) { %>
    <label i18n:translate="label_language">Language</label>
    <select class="form-control" name="language">
      <% _.each(data.languages, function (lang) { %>
        <option value="<%= lang.value %>"><%= lang.title %></option>
      <% }); %>
    </select>
  <% } %>
[ale@flo chameleon]$ cat test.py 
from chameleon import PageTemplate
from zope.i18n import Message

foo = Message("foo", default="Foo", domain="bar")

with open("c.pt") as f:
    pt = PageTemplate(f.read())
    print(pt(foo=foo))
[ale@flo chameleon]$ ./py3/bin/python test.py 
  <% if (data.languages) { %>
    <label>Language</label>
    <select class="form-control" name="language">
      <% _.each(data.languages, function (lang) { %>
        <option value="<%= lang.value %>"><%= lang.title %></option>
      <% }); %>
    </select>
  <% } %>
```

For the record if you had `<?` instead of `<%` zpretty would have not changed it:
```
[ale@flo chameleon]$ cat c.pt 
  <? if (data.languages) { ?>
    <label i18n:translate="label_language">Language</label>
    <select class="form-control" name="language">
      <? _.each(data.languages, function (lang) { ?>
        <option value="<?= lang.value ?>"><?= lang.title ?></option>
      <? }); ?>
    </select>
  <? } ?>
[ale@flo chameleon]$ zpretty c.pt 

<? if (data.languages) { ?>
<label i18n:translate="label_language">Language</label>
<select class="form-control"
        name="language"
>
  <? _.each(data.languages, function (lang) { ?>
  <option value="&lt;?= lang.value ?&gt;"><?= lang.title ?></option>
  <? }); ?>
</select>
<? } ?>
```

Reply:
Also [zptlint](https://pypi.org/project/zptlint/) complains about that file...
```
[ale@flo test_zpretty]$ zptlint properties.pt 
*** Error in: properties.pt
     zope.tal.htmltalparser.NestingError: Open tags <html>, <body>, <div>, <div>, <header>, <div>, <div>, <div>, <div>, <div>, <div>, <div> do not match close tag </option>
     , at line 561, column 43
```

Reply:
I guess this is template markup  interpreted on the JS side.

Reply:
And it does not work with this change.
![screenshot_2023-03-12_23:52:44_selection](https://user-images.githubusercontent.com/157140/224578781-af426332-dc42-4dd4-bd1a-86e5e092ead6.png)

Reply:
The contents of this template are inserted in the folder contents template in `data-pat-structure`.  And this part with the data languages is for a button that I did not know existed. :-). If see that with the original code it correctly adds a list of languages to choose from, if you have multiple languages available in your site.  It might not be a good idea with `plone.app.multilingual`, but that is beside the point here, and I did not try.

@thet Can you explain where this syntax of `<%` actually comes from, because I have not seen it before.  Is that Chameleon?  Javascript?  Patternslib?